src/python/test/test_dxpy.py

```diff
@@ -1636,11 +1687,16 @@ def test_update(self):
         self.assertEqual(dxworkflow.outputFolder, '/foo')
 
         # update title, summary, description, outputFolder by value
-        dxworkflow.update(title='Title', summary='Summary', description='Description', output_folder='/bar/baz')
+        wf_input = [{'name': 'foo', 'class': 'int'}]
+        wf_output = [{'name': 'bar', 'class': 'int'}]
+        dxworkflow.update(title='Title', summary='Summary', description='Description', output_folder='/bar/baz',
+                          workflow_input_spec=wf_input, workflow_output_spec=wf_output)
         self.assertEqual(dxworkflow.editVersion, 1)
         for metadata in ['title', 'summary', 'description']:
             self.assertEqual(getattr(dxworkflow, metadata), metadata.capitalize())
         self.assertEqual(dxworkflow.outputFolder, '/bar/baz')
+        self.assertEqual(dxworkflow.workflowInputSpec, wf_input)
+        self.assertEqual(dxworkflow.workflowOutputSpec, wf_output)
```

Comment:
(Minor: ensure there is no symmetry broken between ability to describe applet/app output and workflows)